apps/taxes/src/modules/provider-connections/ui/providers-section.tsx

```diff
@@ -0,0 +1,36 @@
+import { TextLink } from "@saleor/apps-ui";
+import { Section } from "../../ui/app-section";
+import { ProvidersList } from "./providers-list";
+
+const Intro = () => {
+  return (
+    <Section.Description
+      title="Tax providers"
+      description={
+        <>
+          Saleor offers two ways of calculating taxes: flat or dynamic rates.
+          <br />
+          <br />
+          Taxes App leverages the dynamic rates by delegating the tax calculation to third-party
+          services.
+          <br />
+          <br />
+          You can read more about how Saleor deals with taxes in{" "}
+          <TextLink newTab href="https://docs.saleor.io/docs/3.x/developer/taxes">
+            our documentation
+          </TextLink>
```

Comment:
1. I think you should wrap text ith <Text> to make proper typography
2. Please don't use `<br/><br/>`, its not a replacement for margins. You can have several `<Text marginBottom={x}>`